app/components/Article/index.tsx

```diff
@@ -101,9 +101,9 @@ export const Article = ({question, glossary}: ArticleProps) => {
   const {title, text, pageid} = question
 
   return (
-    <article className="article-container">
-      <h1 className="teal">{title}</h1>
-      <ArticleMeta {...question} />
+    <article>
+      <h1 className="teal padding-bottom-24">{title}</h1>
+      <ArticleMeta question={question} className="padding-bottom-56"/>
```

Comment:
you could just add this directly to the `ArticleMeta` - it's only used here. This is also fine, though

Reply:
This took me a lot of hard work that is not being appreciated. Ha but no I did realize that but did it anyway for practice.